TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/settings.py

```diff
@@ -28,8 +28,23 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-DEPLOYED_STAGE = os.environ.get('MIGRATION_STAGE')
-ALLOWED_HOSTS = ['migration-console', f'migration-console.migration.{DEPLOYED_STAGE}.local', 'localhost']
+
+def get_allowed_hosts():
+    hosts = os.getenv('API_ALLOWED_HOSTS', '')
+    if not hosts:
+        return ['localhost']
+    # Remove extra spacing and/or quotes
+    hosts = hosts.strip()
+    hosts = hosts.replace('"', '')
+    hosts = hosts.replace('\'', '')
+
+    host_list = []
+    for host in hosts.split(','):
+        host_list.append(host.strip())
+    return host_list
```

Comment:
This is minor, but we're doing some unnecessary stuff here--the initial strip doesn't matter because we're stripping the individual items (and that's the level we want to do it at)--and we can do this more directly/concisely. On the other hand, I know that I love python one-liners more than the average person, so feel free to disregard this if it makes it less intuitive. (I did make it a two-liner instead of a one-liner though)

```suggestion
    hosts = hosts.replace('"', '').replace('\'', '')
    return [host.strip() for host in hosts.split(',')]
```

Reply:
Agree this is cleaner, have updated now